packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -259,127 +266,125 @@ export function SchemaDrivenAnnotationEditor(
               </Box>
             </Alert>
           )}
-          {formData != undefined && (
```

Comment:
Hiding the form until we had `formData` was causing issues in Storybook where stories where we rendered the form with a schema but no entity failed to appear.

I think this condition was part of a workaround for another issue that has since been fixed.